onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -95,13 +96,13 @@ TEST(GraphTransformationTests, ConstantFolding) {
   rule_transformer->Register(std::make_unique<ConstantFolding>());
   onnxruntime::GraphTransformerManager graph_transformation_mgr{5};
 
-  graph_transformation_mgr.Register(std::move(rule_transformer));
-  ASSERT_TRUE(graph_transformation_mgr.ApplyAll(graph).IsOK());
+  graph_transformation_mgr.Register(std::move(rule_transformer), TransformerLevel::Level1, {""});
+  ASSERT_TRUE(graph_transformation_mgr.ApplyTransformers(graph, {""}, TransformerLevel::Level1).IsOK());
 
   op_to_count = CountOpsInGraph(graph);
   ASSERT_TRUE(op_to_count["Unsqueeze"] == 0);
 }
-
+/*
 TEST(GraphTransformationTests, FuseConvBNMulAddUnsqueeze) {
```

Comment:
Are these meant to be disabled?

Reply:
Nope - I will enable them before I check-in …. They needed some changes... and I first wanted to finalize the API before I go ahead and change them. Next commit includes them